src/fides/api/api/v1/endpoints/privacy_preference_endpoints.py

```diff
@@ -731,12 +776,17 @@ def save_privacy_preferences(
                 original_request_data=data,
             )
         )
+        saved_preferences_response.tc_mobile_data = convert_tc_string_to_mobile_data(
+            tc_string
```

Comment:
If a TC string was in the request, it is decoded and converted to mobile data format in the response.

Reply:
This is why we had to update the response body for saving Privacy Preferences in the previous PR, to make room for this key.